### PR TITLE
Add draft for create a step template post

### PR DESCRIPTION
**Pull request by ryanrousseau**

### CODE REVIEW

File: blog/2020-06/creating-an-octopus-deploy-step-template/index.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+**Print Command?** - I added this parameter to use "set -x" to print the command when it runs. It comes in handy when debugging and testing the step.
+
+![Parameters for the Firebase Deploy step](firebase-parameters.png)
+
+## Step
+
+With the parameters sorted, I switch to the **Step** tab. My script source will stay set to **Inline source code**. **Script file inside a package** is great if your team has standard scripts stored in a package. It is less suited for a template that you plan to make available to the general public (foreshadowing.)
+
+### Changeup
```
Comment (wordlee) [inline blog/2020-06/creating-an-octopus-deploy-step-template/index.md:91]: I'm not quite sure what this heading is trying to convey (I'm might be missing something).

Comment (ryanrousseau) [reply blog/2020-06/creating-an-octopus-deploy-step-template/index.md:91]: Ah, it meant that I was going out of order in the steps on the screen. Though looking at it now, I guess that doesn't need its own heading.

**Comment by ryanrousseau** [comment]
> 
> 
> Looks good, @ryanrousseau. I left one question on the text. I committed some line edits, as always, nothing major. I changed some of the formatting so that UI labels are bolded and parameters use back tics.

Thanks for the edits @wordlee

**Comment by ryanrousseau** [comment]
I believe this is ready for you now, @robpearson

**Pull request merged**